ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_statistical.py

```diff
@@ -234,3 +234,36 @@ def test_quantile(
         interpolation=interpolation[0],
         keepdims=keep_dims,
     )
+
+
+# nanquantile
+
+
+@handle_test(
+    fn_tree="functional.experimental.nanquantile",
+    dtype_x_axis=statistical_dtype_values(function="quantile"),
+    keepdims=st.booleans(),
+)
+def test_nanquantile(
+    *,
+    dtype_x_axis,
+    keepdims,
+    test_flags,
+    backend_fw,
+    fn_name,
+    ground_truth_backend,
+):
+    input_dtype, x, axis, interpolation, q = dtype_x_axis
+    helpers.test_function(
+        ground_truth_backend=ground_truth_backend,
+        input_dtypes=input_dtype,
+        test_flags=test_flags,
+        fw=backend_fw,
+        fn_name=fn_name,
```

Comment:
You can pass `rtol=1e-03` and `atol=1e-03` as it will be handling the tolerance here for jax uptill 3 decimal spaces.
You can see here how to use these parameters https://github.com/unifyai/ivy/blob/master/ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_elementwise.py#L702